jetty-core/jetty-http-spi/src/test/java/org/eclipse/jetty/http/spi/SPIServerTest.java

```diff
@@ -82,7 +87,244 @@ public void handle(HttpExchange exchange) throws IOException
         });
 
         URL url = new URL("http://localhost:" + port + "/");
-        assertThat(IO.toString(url.openConnection().getInputStream()), is("Hello"));
+        HttpURLConnection http = (HttpURLConnection)url.openConnection();
```

Comment:
We should not be using `HttpURLConnection`.
Either JDK's `HttpClient` or Jetty's, or `Socket` + `HttpTester` or `LocalConnector`.

Reply:
Fixed